docs/topics/install/docker.rst

```diff
@@ -1,111 +1,103 @@
 ====================
-Install via Docker
+Install with Docker
 ====================
 
 .. _install-with-docker:
 
-Want the easiest way to start contributing to AMO? Try our docker-based
+Want the easiest way to start contributing to AMO? Try our Docker-based
 development environment.
 
-First you'll need to install docker_, please check the information for
+First you'll need to install Docker_. Please read their docs for
 the installation steps specific to your operating system.
 
-.. note::
-    Docker recommends installing docker-toolbox_ if you're on OS X or
-    windows and that will provide you with the ``docker-machine`` and
-    ``docker-compose`` (Mac-only).
-
-
-.. _creating-the-docker-vm:
+There are two options for running docker depending on the platform
+you are running.
 
-Creating the docker vm (mac/windows)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ * Run docker on the host machine directly (recommended)
+ * Run docker-machine which will run docker inside a virtual-machine
 
-If you go the docker-machine route your first step is to create a vm::
+Historically Mac and Windows could only run Docker via a vm. That has
+recently changed with the arrival of docker-for-mac_ and docker-for-windows_.
 
-    docker-machine create --driver=virtualbox addons-dev
+If your platform can run Docker directly either on Linux, with docker-for-mac_
+or docker-for-windows_ then this is the easiest way to run it the most minimal
```

Comment:
"then this is the easiest way to run it the most minimal set of requirements"

I think this sentence got tweaked but the tweaking wasn't finished 😄 

"then this is the easiest way to run addons-server" would work.